docs/components/receive.md

```diff
@@ -66,6 +74,13 @@ The example content of `hashring.json`:
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed
 for tenancy and replication.
 
+### Enabling Routing/Ingestion for Thanos Receiver
```

Comment:
I think that `/` can be confusing because it can imply that the things on each side of the slash are the same, which is not the case here.
```suggestion
### Enabling Routing and Ingestion for the Thanos Receiver
```